src/rulesets/Base/arraymath.jl

```diff
@@ -268,3 +268,17 @@ function rrule(::typeof(-), x::AbstractArray)
     end
     return -x, negation_pullback
 end
+
+
+#####
+##### Addition (Multiarg `+`)
+#####
+
+function rrule(::typeof(+), arrs::AbstractArray...)
+    y = +(arrs...)
+    function add_pullback(dy)
+        # using ntuple here for type stability
+        return (NoTangent(), ntuple(i -> dy, length(arrs))...)
+    end
```

Comment:
Not sure it'll matter, but perhaps worth avoiding closing over the arguments, since we only need the length. 
```suggestion
    y = +(arrs...)
    valN = Val(length(arrs)) # makes add_pullback type-stable without closing over arrs
    add_pullback(dy) = (NoTangent(), ntuple(_ -> dy, valN)...)
```